File: src/main/java/com/bot/model/dto/DynamicsDto.java
package com.bot.model.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;

import java.io.File;
import java.math.BigDecimal;
import java.util.List;
import java.util.Map;

@Builder
@Data
@AllArgsConstructor
public class DynamicsDto {

    private String message;
    private BigDecimal totalSpend;
    private File dynamicsFile;
    private Map<String, List<DynamicsDataDto>> monthSpendings;
}

File: plunner-core/src/main/kotlin/com/planner/dto/DynamicsDto.kt
package com.planner.dto

import java.io.File
import java.math.BigDecimal

data class DynamicsDto(
    val message: String,
    val totalSpend: BigDecimal,
    val dynamicsFile: File,
    val monthSpendings: Map<String, List<DynamicsDataDto>> = emptyMap()
)
